test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/IntegrationTests/CodeGenerationIntegrationTest.cs

```diff
@@ -24,6 +23,35 @@ public class CodeGenerationIntegrationTest : IntegrationTestBase
         private static readonly RazorSourceDocument DefaultImports = MvcRazorTemplateEngine.GetDefaultImports();
 
         #region Runtime
+        [Fact]
+        public void Inherits_Runtime()
```

Comment:
This specific test should be in the non-extensions codegen test project. `@inherits` is a core Razor directive. Same with its DesignTime counterpart.